lib/utils/format.ts

```diff
@@ -24,3 +24,38 @@ export const formatPostalCode = (postalCode: string): string => {
 export const formatNumberInput = (numberInputString: string): string => {
   return numberInputString.replace(/^\$/, '');
 };
+
+/**
+ * Format date to be in MM/DD/YYYY format and in UTC time zone to avoid the date being set back by a day
+ * @param {Date} date date to be formatted
+ * @param {boolean} dateInput Whether the date is being displayed in Input element of type date
+ * @returns {string} formatted date
+ */
+export const formatDate = (date: Date, dateInput = false): string => {
+  return dateInput
+    ? new Date(date).toISOString().substring(0, 10)
+    : new Date(date).toLocaleDateString('en-US', { timeZone: 'UTC' });
```

Comment:
I'm guessing this probably doesn't make a difference but should we use 'en-ca' for the locale instead of 'en-us'?

Reply:
en-CA formats it with dashes (MM-DD-YYYY) while en-US formats the dates slashes MM/DD/YYYY. Based on feedback from design we want to use slashes so that's why I opted with en-US